tests/docker-stacks-foundation/test_python_version.py

```diff
@@ -17,8 +17,8 @@ def test_python_version(container: TrackedContainer) -> None:
         tty=True,
         command=["python", "--version"],
     )
-    assert logs.startswith("Python ")
-    full_version = logs.split()[1]
+    python = next(line for line in logs.splitlines() if line.startswith("Python "))
```

Comment:
I think I simplified this line so it stays quite readable and is still robust to logging changes in start.sh